src/_tests_/remove.test.js

```diff
@@ -5,7 +5,7 @@
 import localStorageMock from '../_mocks_/localStorage.js';
 import removeItem from '../_mocks_/remove.js';
```

Comment:
- [ ] <b>REQUIRED:</b> The same here, you are supposed to test the real functions with mocked data.  which means that the `removeItem` function should be imported from `script2.js`. and then use mocked data. Check out this [Article](https://bholmes.dev/blog/mocking-browser-apis-fetch-localstorage-dates-the-easy-way-with-jest/) if you wanted to learn more on how to mock data in localStorage.